bin/cleanup.sh

```diff
@@ -168,7 +168,7 @@ do
 	log4Bash 'DEBUG' "${LINENO}" "${FUNCNAME:-main}" '0' \
 	"check ${ConcordanceID}, ngsId:${ngsId} and arrayId:${arrayId} ngsDnaNo:${ngsDnaNo}, arrayDnaNo:${arrayDnaNo}"
 	
-	if ssh -n "${ATEAMBOTUSER}@${HOSTNAME_PRM}" test -e "/groups/${group}/${PRM_LFS}/concordance/logs/${ConcordanceID}.copyConcordanceCheckData.finished"
+	if ls "${concordanceDir}/logs/${ConcordanceID}"*".ConcordanceCheck.finished" 2>/dev/null
```

Comment:
Is the `*` really required here? If we can do without the *, you can use a proper `test -e` to check if the file exists instead of an `ls`.

Reply:
check